CONTRIBUTING.md

```diff
@@ -0,0 +1,91 @@
+
+- The Version of Rubberduck you are running.
+- Steps to reproduce the issue (Code?).
+- A **full and complete** logfile, preferrably at TRACE-level
+- Screenshots?
+- ...
+
+### Suggesting Enhancements
+
+The team is always happy to hear your ideas. Please do make sure you checked, whether someone else already requested something similar.
+The label [enhancement] will be a good guide.
+
+Do note that some things are just **too large** or **too complex** for Rubberduck to effectively support.
+Please don't be discouraged if your idea gets declined, your feedback is very valuable.
+
+As with Bug Reports the more exact you define what's your gripe, the quicker we can get back to you.
+
+### The Wiki
+
+The Rubberduck Wiki is open for editing to everybody that has a Github account.
+This is very much intentional.
+Everybody is welcome to improve and expand the wiki.
+
+### Coding and Related
+
+So you want to get your hands dirty and fix that obnoxious bug in the last release?
+Well **great**, we like seeing new faces in the fray :)
+Or you just want to find something that you can contribute to to learn C#? Or ...
+Well whatever your motivation is, we recommend taking a look at issues that are labeled [\[up-for-grabs\]](https://github.com/rubberduck-vba/Rubberduck/issues?q=is%3Aissue+is%3Aopen+label%3Aup-for-grabs)
+
+Our Ducky has come quite the long way and is intimidatingly large for a first-time (or even long-time) contributor.
+To make it easier to find something you can do, all issues with that label are additionally labelled with a difficulty.
+That difficulty is a best-guess by the dev-team how complex or evil an issue will be.
+
+If you're new to C#, we recommend you start with \[difficulty-01-duckling\]. The next step \[difficulty-02-ducky\] requires some knowledge about C#, but not so much about how Rubberduck works
+
+Then there's \[difficulty-03-duck\] which requires a good handle on C# and at least an idea of how the components of Rubberduck play toghether.
+And finally there's \[difficulty-04-quackhead\] for when you really want to bang your head against the wall.
+They require both good knowledge of C# and a deep understanding of how Rubberduck works.
+
+In addition to this, a few people have put together some helpful resources about how Rubberduck works internally and help contributors tell left from right in the ~250k Lines of Code.
+
+You can find their work in the [github wiki](https://github.com/rubberduck-vba/Rubberduck/wiki). If something in there doesn't quite seem to match, it's probably because nobody got around to updating it yet. You're very welcome to **improve** the wiki by adding and correcting information. (see right above)
+
+In case this doesn't quite help you or the information you need hasn't been added to the wiki, you can **always** ask questions.
+Please do so in the ["War room"](https://chat.stackexchange.com/rooms/14929).
```

Comment:
I think you actually cannot put the [support] label on an issue unless you are in the dev group.